string file_name
---
uint32 path_number
